src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -98,7 +98,7 @@ public final class InlineConfigParser {
 
     /** A pattern to find the string: "// violation, explanation". */
     private static final Pattern VIOLATION_WITH_EXPLANATION_PATTERN = Pattern
-            .compile(".*//\\s*violation,\\s.+\\s(?:['\"](.*)['\"])?$");
+            .compile(".*//\\s*violation,\\s.+\\s(?:.*)?$");
```

Comment:
Why was this changed? No other PRs so far have had an issue.

Reply:
Nobody noticed defect in this code before until this migration.

Reply:
What was the defect?

Reply:
Before it assumed that there should be quotation marks around the explanation just like it would be around violation's message

Reply:
It was copy paste mistake, fortunately no much usage of this violation style in use.
This PR is first actual usage of this style.

Reply:
`".*//\\s*violation,\\s.+\\s(?:.*)?$"` => `".*//\\s*violation,\\s+(?:.*)?$"`

The extra `.*\\s` doesn't really make sense.

Reply:
moved to https://github.com/checkstyle/checkstyle/issues/15256